### PR TITLE
Fix baseline error

### PR DESCRIPTION
**Pull request by JackBoosY**
osg needs to add features example and plugin, I will open a new PR(#10082) to handle.

Related: #10044 #10043 #10042 #10049 #10050 #10055

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
Thanks!

**Comment by PhoebeHui** [comment]
xlnt failed with x86-windows and arm-uwp, it seems a fake issue, passed on locally testing.

**Comment by vicroms** [comment]
It might be a problem with the downloads cache in the CI machines then.

```
Building package xlnt[core]:x86-windows...
Could not locate cached archive: C:\vsts\_work\3\s\archives\1c\1cfdc355df699185185fc38a2656d62e0d3809a3.zip
-- Using cached C:/vsts/_work/3/s/downloads/tfussell-xlnt-v1.4.0.tar.gz
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:99 (message):
  
  File does not have expected hash:

          File path: [ C:/vsts/_work/3/s/downloads/tfussell-xlnt-v1.4.0.tar.gz ]
      Expected hash: [ 110808064e6c07df83a3ab215f540ff2e2388e22ea7acd78e5872249a66011863c2c551ba741f585f127fe9ad944390df7b27fa7d0bd0948ebe58e96dcc28f81 ]
        Actual hash: [ 74abbee97994098fb7d8fd0839929db74fe01b8428f8bdb8edd28340d3b3ed04d4c7d6dd5d886ae766054ff1b0fe9a8275098a1462e7a5146ff09f1cdb063360 ]
```

**Comment by PhoebeHui** [comment]
@vicroms, thanks for taking a look at this issue!
For xlnt, it seems that Jack updated the hash for fixing the issue, and it passed on other platform except x86-windows and arm-uwp, which looks expect the old hash value.

**Comment by PhoebeHui** [comment]
osg affect by dcmtk.

LLVM failed with following failure:
CMake Error at lib/OptRemarks/cmake_install.cmake:36 (file):
  file INSTALL cannot copy file
  "C:/vsts/_work/4/s/buildtrees/llvm/x64-windows-static-rel/lib/LLVMOptRemarks.lib"
  to
  "C:/vsts/_work/4/s/packages/llvm_x64-windows-static/lib/LLVMOptRemarks.lib".
Call Stack (most recent call first):
  lib/cmake_install.cmake:52 (include)
  cmake_install.cmake:56 (include)

File: ports/osg/portfile.cmake [hunk 1]
```diff
@@ -45,13 +45,15 @@ list(APPEND OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_LIBLAS=ON)
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     OPTIONS
+        -DOSG_USE_UTF8_FILENAME=ON
         -DOSG_USE_UTF8_FILENAME=${OSG_USE_UTF8_FILENAME}
```
Comment (Neumann-A) [inline ports/osg/portfile.cmake]: This does not seem right. doubling of CMake variable

File: ports/qt5-tools/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Source: qt5-tools
-Version: 5.12.5-3
+Version: 5.12.5-4
```
Comment (Neumann-A) [inline ports/qt5-tools/CONTROL]: If you have build failures in qt5-tools it is probably a hidden dependency in qt5-base. You can check the *.prl files for the required dependencies.

Comment (JackBoosY) [reply ports/qt5-tools/CONTROL]: `LINK : fatal error LNK1104: cannot open file 'C:\vsts\_work\4\s\installed\x64-windows-static\debug\lib\Qt5Cored.lib'`

This should be a fake error.

**Comment by JackBoosY** [comment]
@ras0219-msft 
```
C:/vsts/_work/3/s/buildtrees/llvm/x64-windows-static-rel/lib/clangToolingASTDiff.lib
```
```
LINK : fatal error LNK1104: cannot open file 'C:\vsts\_work\4\s\installed\x64-windows-static\debug\lib\Qt5Cored.lib'
```

I can't repro it on my machine, could you take a look?

**Comment by JackBoosY** [comment]
@ras0219-msft Seems bloked on osx pipeline test.

**Comment by PhoebeHui** [comment]
Only llvm issue left, @vicroms @ras0219-msft, could you help have a look? we could not repro this issue locally.

LLVM failed with following failure:
CMake Error at lib/OptRemarks/cmake_install.cmake:36 (file):
file INSTALL cannot copy file
"C:/vsts/_work/4/s/buildtrees/llvm/x64-windows-static-rel/lib/LLVMOptRemarks.lib"
to
"C:/vsts/_work/4/s/packages/llvm_x64-windows-static/lib/LLVMOptRemarks.lib".
Call Stack (most recent call first):
lib/cmake_install.cmake:52 (include)
cmake_install.cmake:56 (include)

**Comment by vicroms** [comment]
Retrying...

**Comment by vicroms** [comment]
Two hours later

```
-- Using cached C:/vsts/_work/4/s/downloads/llvm-8.0.0.src.tar.xz
-- Extracting source C:/vsts/_work/4/s/downloads/llvm-8.0.0.src.tar.xz
-- Applying patch install-llvm-modules-to-share.patch
-- Applying patch fix-linux-build.patch
-- Using source at C:/vsts/_work/4/s/buildtrees/llvm/src/llvm-8-1465978786
-- Using cached C:/vsts/_work/4/s/downloads/cfe-8.0.0.src.tar.xz
-- Extracting source C:/vsts/_work/4/s/downloads/cfe-8.0.0.src.tar.xz
-- Applying patch fix-build-error.patch
-- Applying patch install-clang-modules-to-share.patch
-- Using source at C:/vsts/_work/4/s/buildtrees/llvm/src/cfe-8-a977195bae
-- Configuring x64-windows-static
-- Building x64-windows-static-dbg
-- Building x64-windows-static-rel
-- Installing: C:/vsts/_work/4/s/packages/llvm_x64-windows-static/share/llvm/copyright
-- Performing post-build validation
-- Performing post-build validation done
```

**Pull request merged**